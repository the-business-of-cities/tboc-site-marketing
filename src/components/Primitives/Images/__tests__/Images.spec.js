import { Image, SmartImage, SecondaryImage, Background } from "../index";
import React from "react";
import { shallow } from 'enzyme';

describe("Image", () => {
  it("renders correctly", () => {
    expect(shallow(<Image />)).toMatchSnapshot();
  });
});

describe("SmartImage", () => {
  it("renders correctly", () => {
    expect(shallow(<SmartImage />)).toMatchSnapshot();
  });
});

describe("SecondaryImage", () => {
  it("renders correctly", () => {
    expect(shallow(<SecondaryImage />)).toMatchSnapshot();
  });
});

describe("Background", () => {
  it("renders correctly", () => {
    expect(shallow(<Background />)).toMatchSnapshot();
  });
});
