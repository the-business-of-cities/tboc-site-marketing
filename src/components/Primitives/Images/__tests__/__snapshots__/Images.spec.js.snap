// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background renders correctly 1`] = `
<div
  className=""
/>
`;

exports[`Image renders correctly 1`] = `
.c0 {
  width: 100%;
  height: auto;
  object-fit: contain;
}

<img
  className="c0"
/>
`;

exports[`SecondaryImage renders correctly 1`] = `
.c0 {
  width: 100%;
  object-fix: contain;
}

<img
  className="c0"
/>
`;

exports[`SmartImage renders correctly 1`] = `
.c0 {
  width: 100%;
  padding-top: 100%;
  background-color: rgba(0,0,0,0.2);
  background-image: url();
  background-size: cover;
  background-position: center center;
  background-repeat: norepeat;
  margin-left: auto;
}

<div
  className="c0"
/>
`;
